Makefile

```diff
@@ -465,11 +465,10 @@ kind: ## Create a kind cluster for Cilium development.
 kind-down: ## Destroy a kind cluster for Cilium development.
 	$(QUIET)./contrib/scripts/kind-down.sh
 
-.PHONY: kind-ready
-kind-ready:
-	@$(ECHO_CHECK) kind is ready...
-	@kind get clusters 2>&1 | grep "No kind clusters found." \
-		&& exit 1 || exit 0
+kind-clustermesh: ## Create two kind clusters for clustermesh development.
```

Comment:
I would suggest to add also a `kind-clustermesh-down` target to tear down the environment.